alpha/declcfg/write.go

```diff
@@ -106,6 +106,26 @@ func (writer *MermaidWriter) WriteChannels(cfg DeclarativeConfig, out io.Writer)
 
 	minEdgePackage := writer.getMinEdgePackage(&cfg)
 
+	depByPackage := sets.Set[string]{}
+	depByChannel := sets.Set[string]{}
+	depByBundle := sets.Set[string]{}
+
+	for _, d := range cfg.Deprecations {
+		for _, e := range d.Entries {
+			switch e.Reference.Schema {
+			case SchemaPackage:
+				depByPackage.Insert(d.Package)
+			case SchemaChannel:
+				depByChannel.Insert(e.Reference.Name)
+			case SchemaBundle:
+				depByBundle.Insert(e.Reference.Name)
+			}
+		}
+	}
+
+	var deprecatedPackage string
```

Comment:
I know this PR is merged already, but the code here presumes that we will only ever have at most one deprecated package.  If the scope of the declcfg includes multiple deprecated packages, does this work?